scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/dfn.yaml

```diff
@@ -357,19 +357,24 @@ DFN-8-1EP_2x3mm_P0.5mm_EP1.4x1.7mm:
   overall_height:
     minimum: 0.8
     maximum: 1
+    nominal: 0.9
   lead_width:
     minimum: 0.2
     maximum: 0.3
+    nominal: 0.25
   lead_len:
     minimum: 0.3
     maximum: 0.5
+    nominal: 0.4
   EP_size_x:
     minimum: 1.5
     maximum: 1.75
+    nominal: 1.7
   EP_size_x_overwrite: 1.7
   EP_size_y:
     minimum: 1.3
-    maximum: 1.55
+    maximum: 1.45
```

Comment:
1.55 was correct before.

Reply:
I am not sure. Is the generator taking the pad size of the IC or the pad size of the footprint?

I realized that 1.55 is the size of the pad on the bottom of the chip, but 1.45 is the maximum size for the pad.

Reply:
Except for EP_size_[xy]_overwrite, all definitions are _package_ dimensions from which the footprint is generated.
BTW, please cross-reference FP and generator PRs.